### PR TITLE
fix model zoo doc

### PR DESCRIPTION
**Pull request by jerrywgz**
Fix the url and backbone names in PaddleDetection Model Zoo.
Fix the doc for RetinaNet.

### CODE REVIEW

File: PaddleCV/PaddleDetection/docs/MODEL_ZOO.md [hunk 1]
```diff
@@ -64,12 +63,14 @@ The backbone models pretrained on ImageNet are available. All backbone models ar
 
 ### RetinaNet
 
-| Backbone     | Size | Lr schd | Box AP | Download  |
-| :----------- | :--: | :-----: | :----: | :-------: |
-| ResNet50-FPN | 300  |  120e   |  36.0  | [model](https://paddlemodels.bj.bcebos.com/object_detection/retinanet_r50_fpn_1x.tar) |
-| ResNet101-FPN | 300  |  120e   |  37.3  | [model](https://paddlemodels.bj.bcebos.com/object_detection/retinanet_r101_fpn_1x.tar) |
+| Backbone      | Img/gpu | Lr schd | Box AP | Download  |
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/docs/MODEL_ZOO.md]: 统一把Img -> Image ?

Comment (jerrywgz) [reply PaddleCV/PaddleDetection/docs/MODEL_ZOO.md]: Done

File: PaddleCV/PaddleDetection/docs/MODEL_ZOO.md [hunk 2]
```diff
@@ -66,21 +65,23 @@ The backbone models pretrained on ImageNet are available. All backbone models ar
 | ResNet34     | 416  |    8    |   120e  |  34.3  | [model](https://paddlemodels.bj.bcebos.com/object_detection/yolov3_r34.tar) |
 | ResNet34     | 320  |    8    |   120e  |  31.4  | [model](https://paddlemodels.bj.bcebos.com/object_detection/yolov3_r34.tar) |
 
-**NOTE**: Yolo v3 trained in 8 GPU with total batch size as 64. Yolo v3 training data augmentations: mixup image, 
+**NOTE**: Yolo v3 trained in 8 GPU with total batch size as 64. Yolo v3 training data augmentations: mixup image,
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/docs/MODEL_ZOO.md]: mixup image -> mixup
random distort -> randomly color distortion.
random crop -> randomly cropping
 random interpolate ->  randomly interpolation method 
random flip image -> randomly flipping

Comment (jerrywgz) [reply PaddleCV/PaddleDetection/docs/MODEL_ZOO.md]: Done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**